chains/solana/contracts/programs/ccip-router/src/instructions/v1/offramp.rs

```diff
@@ -877,16 +859,13 @@ mod tests {
             extra_args: SVMExtraArgs {
                 compute_units: 1000,
                 is_writable_bitmap: 1,
-                accounts: vec![
```

Comment:
I'd definitely add the accounts to the hash test as well so we can still have the comparision between go and rust.
Probably the `hash` method should receive the accounts as another param, so it can serialize it here?